SR2023/2023-03-15-competition-details..md

```diff
@@ -0,0 +1,66 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Competition Details
+---
+
+Hello all!
+
+It's only 2 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2023/competition/)!
+
+Before then, we have a few important updates and reminders for you:
+
+## Media Consent
```

Comment:
While we understand this term, is it one that the teams will know?
Perhaps "photos & videos" or something?